configs/predict.yaml

```diff
@@ -5,14 +5,14 @@ defaults:
   - _self_
   - dataset: conll2003
   - serializer: json
+  - hydra: default.yaml
+  - paths: default.yaml
 
   # enable color logging
   - override hydra/hydra_logging: colorlog
   - override hydra/job_logging: colorlog
```

Comment:
is this already defined in `hydra/defaults.yaml`, isn't it?

Reply:
fixed in [e884f23](https://github.com/ChristophAlt/pytorch-ie-hydra-template/pull/42/commits/e884f233ad799e3bbebd68b922e6570403f7b773)